src/main/java/com/sap/oss/phosphor/fosstars/model/value/SemanticVersion.java

```diff
@@ -0,0 +1,81 @@
+package com.sap.oss.phosphor.fosstars.model.value;
+
+import java.util.Objects;
+import java.util.Optional;
+
+public class SemanticVersion {
```

Comment:
Please add javadoc for all classes, fields, finals, methods and constructors. It doesn't have to be long descriptions but at least a simple note.